sstables/sstables.cc

```diff
@@ -1588,6 +1603,7 @@ future<> sstable::load(const dht::sharder& sharder, sstable_open_config cfg) noe
                 std::vector<unsigned>{this_shard_id()} : compute_shards_for_this_sstable(sharder);
     }
     co_await open_data(cfg);
+    _stats.on_seal_or_load(co_await bytes_on_disk_uncached());
 }
```

Comment:
I think we don't want to bump it here, since a sstable can be loaded at one shard, and will be transferred to another shard during boot (without unlink).

Reply:
Unfortunately, if I remove this call, then the stat underflows when unlinking subtracts the sizes of the files being removed.

If I keep this call, then after scylla startup, `http://localhost:9180/metrics` reports
~~~
# HELP scylla_sstables_bytes_on_disk Bytes currently used on disk by sstables
# TYPE scylla_sstables_bytes_on_disk gauge
scylla_sstables_bytes_on_disk{shard="0"} 455755.000000
~~~
and 
~~~
find $SCYLLA_HOME/data/ -type f -printf '%s\n' | (X=0; while read Y; do X=$((X+Y)); done; echo $X)
~~~
also prints 455755 precisely.